airflow/providers/amazon/aws/operators/eks.py

```diff
@@ -401,13 +417,31 @@ def execute(self, context: Context):
             selectors=self.selectors,
             **self.create_fargate_profile_kwargs,
         )
-
-        if self.wait_for_completion:
+        if self.deferrable:
+            self.defer(
+                trigger=EksCreateFargateProfileTrigger(
+                    cluster_name=self.cluster_name,
+                    fargate_profile_name=self.fargate_profile_name,
+                    aws_conn_id=self.aws_conn_id,
+                    poll_interval=self.waiter_delay,
+                    max_attempts=self.waiter_max_attempts,
+                ),
+                method_name="execute_complete",
+                timeout=timedelta(seconds=(self.waiter_max_attempts * self.waiter_delay + 60)),
```

Comment:
why 60 ? Maybe create a named constant ?

Reply:
I don't know if we need a named constant for this. 60 is arbitrary - I think adding a new constant will just add another variable someone will need to look up the value of.

Reply:
how about a comment then ?
```suggestion
                # adding 60s to theoretical value to have extra room to account for time lost here and there
                timeout=timedelta(seconds=(self.waiter_max_attempts * self.waiter_delay + 60)),
```

Reply:
I have a comment that I used in the redshift operators. I'll use that to be consistent:

```
# timeout is set to ensure that if a trigger dies, the timeout does not restart
# 60 seconds is added to allow the trigger to exit gracefully (i.e. yield TriggerEvent)
```